libraries/provider_systemd_unit.rb

```diff
@@ -17,6 +17,17 @@ def whyrun_supported?
     action :create do
       unit_path = Systemd::Helpers.unit_path(new_resource)
 
+      if new_resource.drop_in && new_resource.override.nil?
+        fail(
+          Chef::Exceptions::ValidationFailed,
+          'Required argument override is missing!'
+        )
+      end
```

Comment:
@jkeiser is there a way to use such validation if we inherit directly from lwrpbase? 
@nathwill  we can not we attach any arbitrary proc/lambda as validation in the resource? is it because of the override.nil?